f8a_report/helpers/report_helper.py

```diff
@@ -507,6 +507,12 @@ def normalize_worker_data(self, start_date, end_date, stack_data, worker,
             else:
                 avg_response_time['pypi'] = 0
 
+            if total_stack_requests['golang'] > 0:
+                avg_response_time['golang'] = \
+                    total_response_time['golang'] / total_stack_requests['golang']
+            else:
```

Comment:
Can this be made a method instead of adding on for each ecosystem.

Reply:
Ack!